src/scalarstats.jl

```diff
@@ -660,7 +660,7 @@ function summarystats(a::AbstractArray{T}) where T<:Union{Real,Missing}
     R = typeof(m)
     n = length(a)
     ns = length(s)
-    qs = if m == 0 || n == 0
+    qs = if n == 0
```

Comment:
BTW, this should probably be `ns` rather than `n`, if you feel like fixing another bug I introduced in that function... The difference should be visible when passing only missing values.